src/scripts/destinyTrackerApi/reviewDataCache.js

```diff
@@ -1,31 +1,41 @@
 import _ from 'underscore';
-import { itemTransformer } from './itemTransformer.js';
-
-class reviewDataCache {
+import { ItemTransformer } from './itemTransformer';
+
+/**
+ * Cache of review data.
+ * Mixes and matches remote as well as local data to cut down on chatter and prevent data loss on store refreshes.
+ *
+ * @class ReviewDataCache
+ */
+class ReviewDataCache {
   constructor() {
-    this._itemTransformer = new itemTransformer();
+    this._itemTransformer = new ItemTransformer();
     this._itemStores = [];
   }
 
-  getMatchingItem(item) {
+  _getMatchingItem(item) {
     var dtrItem = this._itemTransformer.translateToDtrWeapon(item);
 
+    // The DTR API isn't consistent about returning reference ID as an int in its responses
+    // and findWhere considers 123 !== "123".
     dtrItem.referenceId = String(dtrItem.referenceId);
 
     return _.findWhere(this._itemStores, { referenceId: dtrItem.referenceId, roll: dtrItem.roll });
   }
 
+  /**
+   * Do we have any locally-cached review data for the given item from the DIM store?
+   *
+   * @param {any} item
+   * @returns {any}
```

Comment:
I saw the note on getMatchingItem() - it's not in translateToDtrWeapon because their API calls for reference IDs to be ints, it's just returning a string in some cases (which I've let them know about).